presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestDistributedQueries.java

```diff
@@ -745,6 +746,61 @@ public void testViewMetadata()
         assertUpdate("DROP VIEW meta_test_view");
     }
 
+    @Test
+    public void testCurrentUserInView()
+    {
+        skipTestUnless(supportsViews());
+        if (!getSession().getCatalog().isPresent() || !getSession().getSchema().isPresent()) {
```

Comment:
When does this happen? We should probably throw `SkipException` here to make it clear the test is skipped due to requirements not met